bridges/ThingiverseBridge.php

```diff
@@ -0,0 +1,76 @@
+<?php
+class ThingiverseBridge extends BridgeAbstract {
+
+	const NAME = 'Thingiverse Search';
+	const URI = 'https://thingiverse.com';
+	const DESCRIPTION = 'Returns feeds for search results';
+	const MAINTAINER = 'AntoineTurmel';
+	const PARAMETERS = array(
+		array(
+			'query' => array(
+				'name' => 'Search query',
+				'type' => 'text',
+				'required' => true,
+				'title' => 'Insert your search term here',
+				'exampleValue' => 'Enter your search term'
+			),
+			'queryextension' => array(
+				'name' => 'Query extension',
+				'type' => 'text',
+				'requied' => false,
+				'title' => 'Insert additional query parts here
```

Comment:
I am not really found of asking the user to supply manually all the parameters, though there is no current way of excluding parameters depending on a list...
At least I would recommend you allowing the category as a parameter, as this is useful and quite straightforward.